Ladybird/AppKit/main.mm

```diff
@@ -22,6 +22,9 @@
 #import <UI/Tab.h>
 #import <UI/TabController.h>
 
+#include <Ladybird/MachPortServer.h>
+static OwnPtr<Ladybird::MachPortServer> s_mach_port_server;
```

Comment:
Does this need to be declared here? Doesn't seem like anything uses it outside of `main()`

Reply:
(same question on the Qt side)

Reply:
Nope it doesn't, moved ownership to main.